src/main/java/carbon/Storage.java

```diff
@@ -0,0 +1,73 @@
+package carbon;
+
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.util.Scanner;
+
+/**
+ * Stores data on tasks and acts as interface for storage.
+ * Used to save data and load data from a default filepath.
+ */
+public class Storage {
+    /** Default filepath for storing data on tasks */
+    private static final String WORKING_DIR = System.getProperty("user.dir");
+    private static final String SAVEFILE = Storage.WORKING_DIR + "/../../../store/tasks.txt";
+    private static final String SAVEFILE_DIR = Storage.WORKING_DIR + "/../../../store/";
+
+    /**
+     * Constructs an instance of Storage class.
+     *
+     * @return Storage object.
+     */
+    public Storage() {}
+
+    /**
+     * Loads the savefile and retrieves saved tasks.
+     * Uses the default filepath.
+     *
+     * @return TaskList instance containing the saved tasks.
+     */
+    public TaskList loadSavefile() {
```

Comment:
Could make it Camel case? Because Savefile could be the name of the file but loadSaveFile looks better

Reply:
Oh yeah whoops, I have a habit of writing the term in a single word like "savefile". Thanks for catching that.